.travis.yml

```diff
@@ -39,16 +39,17 @@ android:
   - sys-img-armeabi-v7a-google_apis-21
 
 # Emulator Management: Create, Start and Wait
-#before_script:
-#  - android list targets
-#  - echo no | android create avd --force -n test -t android-21 --abi armeabi-v7a --tag google_apis
-#  - emulator -avd test -no-skin -no-audio -no-window &
+before_script:
+  - android list targets
+  - echo no | android create avd --force -n test -t android-21 --abi google_apis/armeabi-v7a
+  - emulator -avd test -no-window -gpu off -no-boot-anim &
+  - android-wait-for-emulator
+  - travis_wait 20 android-wait-for-emulator
+  - adb shell input keyevent 82 &
```

Comment:
a comment saying what `82` means will be good, so w. don't change it by accident